docs/getting-started-sandbox.md

```diff
@@ -0,0 +1,91 @@
+---
+id: getting-started-sandbox
+title: Getting Started With PayID Sandbox
+sidebar_label: Getting Started With PayID Sandbox
+---
+
+With the [PayID Sandbox](https://payid.org/sandbox), you can experiment with PayID without deploying your own server. Although the PayID Sandbox can be used to create PayIDs, do not use these PayIDs that rely on this virtual server to send or receive real cryptocurrency. The PayID Sandbox does not directly support Verifiable PayID.
+
+You can also experiment with PayID by using [PayID CLI](payid-cli), which does support Verifiable PayID.
+
+The sandbox allows you to experiment with PayID, but does not create an application that allows you to send or receive currency using the PayID.
+
+You are encouraged to copy the cURL or JavaScript code snippets that the sandbox provides and adapt them to your app.
+
+Requirements:
+
+- GitHub account. You will use this GitHub account to log in to PayID Sandbox.
+- Crypto addresses on other wallets or exchanges. You can map your PayID to these crypto addresses.
+
+## Open PayID Sandbox and Create a User
+
+1. Open [PayID Sandbox](https://payid.org/sandbox) and log in with your GitHub account.
+   The sandbox sets up a virtual server for you at _github_user_name_.sandbox.payid.org.
```

Comment:
> _github_user_name_.sandbox.payid.org

Trivial but consider placing all of the above in code format and making the variable obnoxiously obvious, i.e.:

> `<YOUR_GITHUB_USERNAME>.sandbox.payid.org`

Reply:
Done.